core/src/main/scala/chisel3/Aggregate.scala

```diff
@@ -937,6 +937,28 @@ abstract class Record(private[chisel3] implicit val compileOptions: CompileOptio
     case _ => false
   }
 
+  trait AutoCloneType { self: Record =>
+    override def cloneType: this.type = _cloneTypeImpl.asInstanceOf[this.type]
+  }
+
+  private def checkClone(clone: Record): Unit = {
+    for ((name, field) <- elements) {
+      if (clone.elements(name) eq field) {
+        throw new AutoClonetypeException(
+          s"The bundle plugin was unable to clone $clone that has field '$name' aliased with base $this." +
+            " Try wrapping the field(s) in Input(...), Output(...), or Flipped(...) if appropriate." +
+            " As a last resort, you can override cloneType manually."
```

Comment:
```suggestion
          s"The compiler plugin was unable to clone $clone that has field '$name' aliased with base $this." +
            " Try wrapping the field(s) in Input(...), Output(...), or Flipped(...) if appropriate."
```
It might also be worth suggesting something like "It is best to avoid nesting any Data arguments inside of other data structures" and something like "as a last resort, you can call chisel3.reflect.DataMirror.internal.chiselTypeClone on any nested Data arguments". 

Something like that, but please massage the phrasing 🙂